qtawesome/styles.py

```diff
@@ -0,0 +1,122 @@
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+"""
+
+from qtpy.QtGui import QPalette, QColor
+from qtpy.QtCore import Qt
+
+
+def dark(app):
+    """ Apply Dark Theme to the Qt application instance.
+
+        Args:
+            app (QApplication): QApplication instance.
+    """
+
+    darkPalette = QPalette()
+
+    # base
+    darkPalette.setColor(QPalette.WindowText, QColor(180, 180, 180))
+    darkPalette.setColor(QPalette.Button, QColor(53, 53, 53))
+    darkPalette.setColor(QPalette.Light, QColor(180, 180, 180))
+    darkPalette.setColor(QPalette.Midlight, QColor(90, 90, 90))
+    darkPalette.setColor(QPalette.Dark, QColor(35, 35, 35))
+    darkPalette.setColor(QPalette.Text, QColor(180, 180, 180))
+    darkPalette.setColor(QPalette.BrightText, QColor(180, 180, 180))
+    darkPalette.setColor(QPalette.ButtonText, QColor(180, 180, 180))
+    darkPalette.setColor(QPalette.Base, QColor(42, 42, 42))
+    darkPalette.setColor(QPalette.Window, QColor(53, 53, 53))
+    darkPalette.setColor(QPalette.Shadow, QColor(20, 20, 20))
+    darkPalette.setColor(QPalette.Highlight, QColor(42, 130, 218))
+    darkPalette.setColor(QPalette.HighlightedText, QColor(180, 180, 180))
+    darkPalette.setColor(QPalette.Link, QColor(56, 252, 196))
+    darkPalette.setColor(QPalette.AlternateBase, QColor(66, 66, 66))
+    darkPalette.setColor(QPalette.ToolTipBase, QColor(53, 53, 53))
+    darkPalette.setColor(QPalette.ToolTipText, QColor(180, 180, 180))
+    darkPalette.setColor(QPalette.LinkVisited, QColor(80, 80, 80))
+
+    # disabled
+    darkPalette.setColor(QPalette.Disabled, QPalette.WindowText,
+                         QColor(127, 127, 127))
+    darkPalette.setColor(QPalette.Disabled, QPalette.Text,
+                         QColor(127, 127, 127))
+    darkPalette.setColor(QPalette.Disabled, QPalette.ButtonText,
+                         QColor(127, 127, 127))
+    darkPalette.setColor(QPalette.Disabled, QPalette.Highlight,
+                         QColor(80, 80, 80))
+    darkPalette.setColor(QPalette.Disabled, QPalette.HighlightedText,
+                         QColor(127, 127, 127))
+
+    app.style().unpolish(app)
+    app.setPalette(darkPalette)
+
+    app.setStyle('Fusion')
+
+
+def light(app):
+    """ Apply Light Theme to the Qt application instance.
+
+        Args:
+            app (QApplication): QApplication instance.
+    """
+
+    lightPalette = QPalette()
```

Comment:
```suggestion
    light_palette = QPalette()
```

Again, let's reserve the camel case usage for classes. Please also change this name below.